pom.xml

```diff
@@ -90,9 +90,9 @@
                 </dependency>
                 
                 <dependency>
-                        <groupId>com.github.samtools</groupId>
-                        <artifactId>htsjdk</artifactId>
-                        <version>2.13.1-2-g46e6590-SNAPSHOT</version>
+                        <groupId>eu.elixir.ega.ebi</groupId>
+                        <artifactId>ega-htsjdk</artifactId>
```

Comment:
This version currently breaks the VCF htsget API: it becomes impossible to open a VCFFileReader with HTTP-based URLs. This needs to be addressed in ega-htsjdk first, before this can be added to DataEdge again.